prog/vm/host_nexus.rb

```diff
@@ -98,7 +98,7 @@ def before_run
   end
 
   label def setup_spdk
-    bud Prog::SetupSpdk
```

Comment:
Seems like the introduction of the storage directory is glutting the diffs here.